ml-proto/test/stdio_write.wast

```diff
@@ -0,0 +1,14 @@
+(module 
+  (import $write "stdio" "write" (param i32 i32))
+
+  (memory 4096 4096 (segment 0 "hello, world!\0D\0A\00"))
```

Comment:
It is very useful to be able to run the testcases by hand, with output going to the console, and if the Windows console needs us to print a `\r`, then that's a problem we need to solve.